### ISSUE TITLE
Loading of images via 2G generally fails

### ISSUE THREAD

**Issue by krombel** [vector-im/riot-android#1486]
In most cases it is not possible to view images when trying to load them via 2G. I think because the net is not fast enough to load the image in <Timeout> seconds. Badly there is then no possibility to rerequest them afterwards. Can there be introduced the option to rerequest the image loading when it failed previously?

**Comment by ylecollen** [comment]
@krombel 
i improved the request timeouts management : their values are set according to the network speed
https://github.com/matrix-org/matrix-android-sdk/commit/7b94d00da7f3d248161a4888d22b49758a248683

Could you test if the application behaviour seems better on your side ?

**Comment by ylecollen** [comment]
You're right we need to add a button to retry the download (by now, if you put the app in background and debackground it, the image download will be retry)
